WORKING-GROUPS.md

```diff
@@ -2,25 +2,26 @@
 
 ## What are Working Groups
 
-
 Working Groups are a way for Ansible community members to self-organize around particular topics of interest.
 
 The basic components of a working group:
-* A group name and charter (why the group exists);
-* A registered IRC channel on [irc.libera.chat](https://libera.chat/) (will usually be #ansible-groupname);
-* A group of users (at least two!) who will be driving the agenda of the working group;
+* Group name and charter (why the group exists);
+* Registered [real-time chat](https://docs.ansible.com/ansible/devel/community/communication.html#real-time-chat) channel;
+* Group of users (at least two!) who will be driving the agenda of the working group;
 * Dedicated wiki space.
 
 The basic responsibilities of a working group:
-* Be responsive on your IRC channel;
+* Be responsive on your real-time chat channel;
 * Report semi-regularly on the cool stuff that your working group is working on;
 * Keep your wiki space clean!
```

Comment:
Can you also please include the following?

```
* Make sure to follow the [Ansible Community Code of Conduct (https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
```

Reply:
Makes sense to mention here, thanks, done